tests/regression/roles/sonic_tacacs_server/defaults/main.yml

```
@@ -85,7 +85,67 @@ tests:
             timeout: 14
             priority: 4
 
-test_delete_all:
   - name: test_case_06
+    description: replace some parameter of tacacs servers
+    state: replaced
+    input:
+      auth_type: mschap
+      source_interface: "{{ interface3 }}"
+      timeout: 36
+      servers:
+        host:
+          - name: my_host
+            auth_type: chap
+            port: 55
+            timeout: 12
+            priority: 3
+
+  - name: test_case_07
+    description: replace hosts of tacacs servers
+    state: replaced
+    input:
+      auth_type: mschap
+      source_interface: "{{ interface3 }}"
+      timeout: 36
+      servers:
+        host:
+          - name: my_host
+            auth_type: chap
+            port: 55
+            timeout: 12
+            priority: 3
+          - name: 20.21.22.23
+            auth_type: login
+            port: 50
+            timeout: 38
+            priority: 4
+          - name: 18.21.22.23
+            auth_type: chap
+            port: 20
+            timeout: 19
+            priority: 8
+
+  - name: test_case_08
+    description: overridden parameter of tacacs servers
```

```suggestion
    description: Override parameter of tacacs servers
```
Please capitalize the test case descriptions.

---

Changed. Thanks!